### PR TITLE
fix(error stacktrace): add stacktrace data to errors

### PR DESCRIPTION
**Pull request by nemoshlag**

### CODE REVIEW

**Comment by nemoshlag** [comment]
1) pass data between spans without corrupting attributes 
2) add error type and error stacktrace to span
3) simplified the errorHandler function in exporter.js

File: packages/web/src/exporter.js [hunk 1]
```diff
@@ -175,32 +178,29 @@ class EpsagonExporter extends CollectorTraceExporter {
         rootSubList[rootSubPos].events.unshift({
           name: rootType.EXCEPTION,
           attributes: [
-            { key: spanAttributeNames.EXCEPTION_MESSAGE, value: { stringValue: err } },
+            { key: spanAttributeNames.EXCEPTION_MESSAGE, value: { stringValue: err.message } },
+            { key: spanAttributeNames.EXCEPTION_TYPE, value: { stringValue: err.type || err.message } },
```
Comment (osherv) [inline packages/web/src/exporter.js:182]: Maybe default could be 'Exception' or something

File: packages/web/src/exporter.js [hunk 2]
```diff
@@ -175,32 +178,29 @@ class EpsagonExporter extends CollectorTraceExporter {
...
-      const spanErrs = [];
       const finalSpans = [];
       spansList[rootSpan.doc.position].spans.forEach((span) => {
-        if (span.name === rootType.ERROR && !spanErrs.includes(EpsagonUtils.getFirstAttribute(span).value.stringValue)) {
-          const errAttr = span.attributes.filter((attr) => attr.key === spanAttributeNames.MESSAGE);
-          const spanStringError = errAttr && errAttr.length ? errAttr[0].value.stringValue : rootType.EXCEPTION;
-
-          spanErrs.push(spanStringError);
+        if (span.name === rootType.ERROR) {
+          const errData = errSpan.filter((s) => s.traceID === span.traceID);
```
Comment (osherv) [inline packages/web/src/exporter.js:194]: so err or error?

File: packages/web/src/instrumentation/documentLoadInstrumentation.js [hunk 3]
```diff
@@ -64,9 +64,16 @@ class EpsagonDocumentLoadInstrumentation extends DocumentLoadInstrumentation {
...
         type: 'browser',
         operation: 'page_load',
-        // "stack": stack
       },
     }, this.epsParentSpan.currentSpan ? api.trace.setSpan(api.context.active(), this.epsParentSpan.currentSpan) : undefined);
+    span.exceptionData = {
+      name: 'exception',
+      attributes: [
+        { key: 'exception.message', value: { stringValue: error.message } },
+        { key: 'exception.type', value: { stringValue: error.type || error.message } },
```
Comment (osherv) [inline packages/web/src/instrumentation/documentLoadInstrumentation.js:73]: same same ^^

**Review by osherv — APPROVED** [review]

**Pull request merged**